### PR TITLE
Abbadie119

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**

### CODE REVIEW

File: ParisBNF/abb/BNFabb119.xml [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+                                <locus from="1r"/>
+                                <title type="complete" ref="LIT3515Epistle">Epistle to the Romans</title>
+                                <textLang mainLang="gez"/>
+
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="1r"/>
+                                    <title type="complete" ref="LIT3515Epistle#Introduction"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                </msItem>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:54]: delete the closing /msItem to correct nesting, 1.1.1 and 1.1.2 should be nested together under 1.1

Comment (eu-genia) [reply ParisBNF/abb/BNFabb119.xml:54]: the closing /msItem should be added instead before 1.2 (after 1.15.1)

File: ParisBNF/abb/BNFabb119.xml [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+                                    <title type="incomplete" ref="LIT3518Epistle">Epistle to the Galathians</title>
+                                    <textLang mainLang="gez"/>
+
+                                    <msItem xml:id="ms_i1.15.1">
+                                        <locus target="#87v"/>
+                                        <title type="complete" ref="LIT3518Epistle#Introduction">Epistle to the Galathians</title>
+                                        <textLang mainLang="gez"/>
+                                      </msItem>
+                                      </msItem>
+
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:231]: add `</msItem>` here

File: ParisBNF/abb/BNFabb119.xml [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+                                  <textLang mainLang="gez"/>
+
+                                  <msItem xml:id="ms_i3.6.1">
+                                      <locus target="#127"/>
+                                      <title type="complete" ref="LIT3528Epistle#Introduction">Epistle to Titus</title>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i1.3.7">
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:322]: should be 3.7 (not 1.3.7)

File: ParisBNF/abb/BNFabb119.xml [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+                                    <title type="complete" ref="LIT3524Epistle">Epistle to the Hebrews</title>
+                                    <textLang mainLang="gez"/>
+
+                                    <msItem xml:id="ms_i1.3.7.1">
+                                        <locus target="#127"/>
+                                        <title type="complete" ref="LIT3524Epistle#Introduction">Epistle to the Hebrews</title>
+                                        <textLang mainLang="gez"/>
+                                </msItem>
+                                  </msItem>
+
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:333]: add `</msItem>` here to close number 3

File: ParisBNF/abb/BNFabb119.xml [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i1.3.7">
+                                    <locus target="#127"/>
+                                    <title type="complete" ref="LIT3524Epistle">Epistle to the Hebrews</title>
+                                    <textLang mainLang="gez"/>
+
+                                    <msItem xml:id="ms_i1.3.7.1">
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:327]: should be 3.7.1

File: ParisBNF/abb/BNFabb119.xml [hunk 6]
```diff
@@ -0,0 +1,547 @@
...
+                            <locus target="#150r"/>
+                            <title type="incomplete">Epistle to Seven Churches, further non-identified</title>
+                          </msItem>
+
+                          <msItem xml:id="ms_i6">
+                        <locus target="#150r"/>
+                        <title type="complete">Another epistle, further non-identified</title>
+
+                      </msItem>
+                              </msItem>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:349]: delete /msItem

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb119.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml:322]: ok

File: ParisBNF/abb/BNFabb119.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml:333]: ok

File: ParisBNF/abb/BNFabb119.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml:327]: ok

File: ParisBNF/abb/BNFabb119.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml:349]: ok

File: ParisBNF/abb/BNFabb119.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml:231]: ok

Comment (eu-genia) [reply ParisBNF/abb/BNFabb119.xml:233]: Actes (line 234) -> correct to Acts

File: ParisBNF/abb/BNFabb119.xml [hunk 7]
```diff
@@ -319,12 +322,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </msItem>
                                 </msItem>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml:323]: delete /msItem here, move it before ms_i4

**Review by eu-genia — CHANGES_REQUESTED** [review]
still some nesting problem

File: ParisBNF/abb/BNFabb119.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml]: I think I did it.

File: ParisBNF/abb/BNFabb119.xml [hunk 8]
```diff
@@ -320,7 +320,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                       <title type="complete" ref="LIT3528Epistle#Introduction">Epistle to Titus</title>
                                     <textLang mainLang="gez"/>
                                 </msItem>
-                                </msItem>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb119.xml]: still nesting issue - remove one /msItem before 3.6 (line 311) and insert one /msItem before 3.7

**Review by eu-genia — CHANGES_REQUESTED** [review]
still nesting issue - remove one /msItem before 3.6 (line 311) and insert one /msItem before 3.7

File: ParisBNF/abb/BNFabb119.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb119.xml]: done

**Review by eu-genia — APPROVED** [review]

**Pull request merged**